### DISCUSSION TITLE
HRT Intl

### DISCUSSION THREAD

**Discussion by darrencycheung** [iptv-org/community#820] [Q&A]
### Channel ID

HRTInternational.hr

### Notes

Where can I get the m3u8?

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Alstruit** [comment]
DRM'd.
https://cdn1-hrt.cdn.eurovisioncdn.net/hrtliveorigin/hrt5.smil/1/manifest.mpd

**Comment by AntiPontifex** [comment]
Dead issue: please close.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19776
